patterns/1-initial/cross-team-retrospectives.md

```diff
@@ -0,0 +1,75 @@
+## Title
+
+Retrospectives for continuous improvement
+
+## Patlet
+
+A host team working with contributors outside of their own line of management constantly runs into misunderstandings.
+As a result collaboration becomes brittle and frustrating.
+Setting aside time for regular retrospectives for the InnerSource team consisting of trusted committers and contributors can help make communication smooth.
+
+## Problem
+
+For long running collaborations friction between host team and collaborators is substantially reducing focus and energy for everyone involved.
+Willingness to continue the collaboration is shrinking.
+
+## Context
+
+A host team of trusted committers has started a long running collaboration with a group of contributors.
+
+* Over time the number of misunderstandings grows.
+* People may run into mis-communication.
+* Teams may discover slight differences in development culture.
+* Team members may discover that assumptions they made about the other team are false.
+* Contribution processes may not be entirely clear and workable for everyone involved.
+
+## Forces
+
+* Participants are inclined to take "written over verbal" as "only written".
+* Trusted committers are all part of the same team.
+* There is a group of contributors all coming from the same team.
+* As a result trusted committers know each other well and understand constraints, prioritization side effects and team dynamics without ever sharing them with contributors.
+* Also contributors form a well knit group.
+* The contribution process is seen as transient and temporary.
+As a result little is invested in forming a shared team of trusted committers and contributors.
+* There is no clear path from contributor to becoming trusted committer - other than becoming a member of the host team.
+
+## Solution
+
+Bring host team and contributors together:
+
+* As a first step it can help to share a meal together and get to know each other.
+* For collaborations running over several weeks establish a monthly 30 minute retrospective meeting that involves everyone who is needed for a successful contribution.
```

Comment:
```suggestion
* For collaborations running over several weeks establish a monthly retrospective meeting that involves everyone who is needed for a successful contribution.
```

Nitpicking:
The list below adds up to 39 minutes.
So maybe to make it easier, we just leave the exact duration away here?

Reply:
Ups :) Fully agree. Thanks for catching this.